src/InboundServer/handlers.js

```diff
@@ -600,6 +646,57 @@ const putTransfersByIdError = async (ctx) => {
     ctx.response.body = '';
 };
 
+/**
+ * Handles PUT /thirdPartyRequests/transactions/{ID} request. 
+ * This is response to a POST /thirdPartyRequests/transactions request
+ */
+const putThirdPartyReqTransactionsById = async (ctx) => {
```

Comment:
can you refactor the code, so there will be no need to copy-paste the test related code?

Reply:
As discussed ignoring it for now